lib/vsc/utils/fancylogger.py

```diff
@@ -512,7 +543,8 @@ def logToUDP(hostname, port=5005, enable=True, datagramhandler=None, name=None):
                            )
 
 
-def _logToSomething(handlerclass, handleropts, loggeroption, enable=True, name=None, handler=None):
+def _logToSomething(handlerclass, handleropts, loggeroption,
+                    enable=True, name=None, handler=None, formatterclass=logging.Formatter):
```

Comment:
Why? The only difference would be in the case where somebody is reassigning or moneky-patching `logging.Formatter